db/data/20210319102418_make_organisations_consistent.rb

```diff
@@ -0,0 +1,23 @@
+# Run me with `rails runner db/data/20210319102418_make_organisations_consistent.rb`
+
+# Put your Ruby code here
+
+service_owner = Organisation.find_by(service_owner: true)
+
+def extending_organisation_for_activity(activity)
+  if activity.level == "programme"
+    activity.children ? activity.children.first.organisation : activity.organisation
+  else
+    activity.parent.organisation
+  end
+end
+
+Activity.where(extending_organisation: nil).each do |activity|
```

Comment:
On production, there are no activities with an empty `extending_organisation` whose form is complete, so not how useful this will be.

I'm wondering whether this needs to be in a separate branch which also includes fixes to the policy files to enforce based on this change?

Reply:
Yeah, I did wonder how useful this would be on reflection. Happy to cherry pick it into another branch.

Reply:
I've cherry picked this commit into a temp branch, so feel free to delete it

Reply:
@pezholio Does the above (second paragraph) seem reasonable..?

Reply:
@leeky "fixes to the policy files to enforce based on this change?" - Is there a story for this?

Reply:
@pezholio Not yet!